### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by molly-moen**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Update README to align with new template

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [x] I have updated the documentation accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,84 +3,181 @@
...
+[Azure IoT Remote Monitoring](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet)
+project to provide a generic implementation for an end-to-end IoT solution.
+More information [here][rm-arch-url].
+
+## Features
+* Create or update device groups
+* Get all or a single device group
+* Get or upload logo
+* Set or get overall solution settings
+* Set or get individual user settings
```
Comment (ppathan) [inline README.md]: nit: `Get or set`

File: README.md [hunk 2]
```diff
@@ -3,84 +3,181 @@
...
-===========================
-## Local Setup
+This service depends on the following repositories.
+Run those services from the instructions in their READMEs in the following order.
 
-### 1. Environment Variables
+1. [Storage Adapter Dotnet Microservice](https://github.com/Azure/pcs-storage-adapter-dotnet)
+1. [Telemetry Dotnet Microservice](https://github.com/Azure/device-telemetry-dotnet)
+1. [IoTHub Manager Dotnet Microservice](https://github.com/Azure/iothub-manager-dotnet)
+1. [Device Simulation Dotnet Microservice](https://github.com/Azure/device-simulation-dotnet)
```
Comment (ppathan) [inline README.md]: Can you open a tracking bug for removing this section once the dependencies have been removed.

File: README.md [hunk 3]
```diff
@@ -3,84 +3,181 @@
...
+[here](#configuration-and-environment-variables).
+   1. `PCS_STORAGEADAPTER_WEBSERVICE_URL` = http://localhost:9022/v1
+   1. `PCS_DEVICESIMULATION_WEBSERVICE_URL` = http://localhost:9003/v1
+   1. `PCS_IOTHUBMANAGER_WEBSERVICE_URL` = http://localhost:9002/v1
+   1. `PCS_TELEMETRY_WEBSERVICE_URL` = http://localhost:9004/v1
+   1. `PCS_AZUREMAPS_KEY` = static
+1. Use the scripts in the [scripts](scripts) folder for many frequent tasks:
+   *  `build`: compile all the projects and run the tests.
+   *  `compile`: compile all the projects.
+   * `run`: compile the projects and run the service. This will prompt for
```
Comment (ppathan) [inline README.md]: nit: add space before `run`

File: README.md [hunk 4]
```diff
@@ -3,84 +3,181 @@
...
-
+## Project Structure
+This microservice contains the following projects:
+* **WebService.csproj** - C# web service exposing REST interface for config functionality
+* **WebService.Test.csproj** - Unit tests for web services functionality
+* **Services.csproj** - C# assembly containining business logic for interacting 
+with storage microserivce, telemetry microservice, device simulation microservice
+and IoTHub manager microservice
+* **Services.Test.csproj** - Unit tests for services functionality
+* **Solution/scripts** - contains build scripts, docker container creation scripts,
```
Comment (ppathan) [inline README.md]: nit: `Contains`

File: README.md (hunk 2 above)
Comment (molly-moen) [reply README.md]: https://github.com/Azure/pcs-config-dotnet/issues/90

**Pull request merged**